src/signals/incident-management/containers/IncidentDetail/types.ts

```diff
@@ -0,0 +1,163 @@
+import type { FetchError } from 'hooks/useFetch';
+
+export interface Context {
+  incident?: Incident;
+  update?: (action: string) => void;
+  preview?: (section: string, payload?: Partial<State>) => void;
+  edit?: (section: string, payload?: Partial<State>) => void;
+  close?: () => void;
+}
+
+export interface State {
+  incident?: Incident;
+  attachments?: Result<Attachment>;
+  children?: Result<IncidentChild>;
+  childrenHistory?: HistoryEntry[][];
+  history?: HistoryEntry[];
+  error?: boolean | FetchError;
+  attachmentHref?: string;
+  patching?: string;
+  edit?: string;
+  preview?: string;
+}
+
+interface Address {
+  postcode: string;
+  huisletter: string | null;
+  huisnummer: string;
+  woonplaats: string;
+  openbare_ruimte: string;
+  huisnummer_toevoeging: string;
+}
+
+type Coordinates = [number, number];
+
+export interface Incident {
+  _links: {
+    curies: { href: string; name: string };
+    _link: { self: { href: string } };
+    archives: { href: string };
+    'sia:children': { href: string }[];
```

Comment:
sia:children and sia:parent should be typed as optional (only present for hoofdmelding / deelmelding respectively)